paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -178,7 +179,18 @@ export const refreshIdsAndPreferences = async ({
 
     // 1. Any Prebid 1st party cookie?
     const strIds = getCookieValue(Cookies.identifiers);
+    const lestRefresh = getCookieValue(Cookies.lastRefresh);
     const strPreferences = getCookieValue(Cookies.preferences);
+    const currentPafData = fromClientCookieValues(strIds, strPreferences);
+
+    const triggerNotification = (freshConsent: boolean) => {
+      const currentlySelectedConsent = currentPafData.preferences?.data?.use_browsing_for_personalization;
+      const shouldShowNotification = !strPreferences || freshConsent !== currentlySelectedConsent;
+
+      if (shouldShowNotification) {
+        showNotification(freshConsent);
```

Comment:
There used to be a check that the consent value is not `undefined`, before showing the notification.
Don't we want to preserve it here?

Reply:
It is an edge case to have consent undefined and to show a notification, I'd say it's impossible. 
But I agree, that an additional check will make it more readable.